pkg/cmd/provisioning/aws/create_all.go

```diff
@@ -0,0 +1,116 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	awssdk "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+
+	"github.com/openshift/cloud-credential-operator/pkg/aws"
+	"github.com/openshift/cloud-credential-operator/pkg/cmd/provisioning"
+)
+
+var (
+	// CreateAllOpts captures the options that affect creation/updating
+	// of the generated objects.
+	CreateAllOpts = options{
+		TargetDir: "",
+	}
+)
+
+func createAllCmd(cmd *cobra.Command, args []string) {
+	cfg := &awssdk.Config{
+		Region: awssdk.String(CreateAllOpts.Region),
+	}
+
+	s, err := session.NewSession(cfg)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	awsClient := aws.NewClientFromSession(s)
+
+	publicKeyPath := CreateAllOpts.PublicKeyPath
+	if publicKeyPath == "" {
+		publicKeyPath = path.Join(CreateAllOpts.TargetDir, publicKeyFile)
+	}
+
+	if err := createKeys(CreateAllOpts.TargetDir); err != nil {
+		log.Fatalf("Failed to create public/private key pair: %s", err)
+	}
+
+	identityProviderARN, err := createIdentityProvider(awsClient, CreateAllOpts.Name, CreateAllOpts.Region, publicKeyPath, CreateAllOpts.TargetDir, false)
+	fmt.Printf("ARN: %+v\n", identityProviderARN)
```

Comment:
Should this be a log output?

Reply:
This is not needed as we already print the created ARN inside `createIdentityProvider` funciton

sample: `2021/04/09 12:18:30 Identity Provider created with ARN: <arn>`